esmvaltool/recipes/cmorizers/recipe_era5-land.yml

```diff
@@ -0,0 +1,32 @@
+# ESMValTool
+# recipe_era5.yml
+---
+documentation:
+  description: CMORize ERA5-Land data
+
+  authors:
+    - andela_bouwe
+    - kalverla_peter
+
+  references:
+    - acknow_project
+
+  projects:
+    - ewatercycle
+
+datasets:
+  # For the daily diagnostic, always add the next year, otherwise the last day is not cmor compatible
+  - {dataset: ERA5-Land, project: native6, type: reanaly, version: '1',
+     tier: 3, start_year: 1981, end_year: 1982}
+
+diagnostics:
+  monthly:
+    description: CMORize monthly ERA5 data
+    scripts:
+      rename:
+        script: cmorizers/era5.py
+    variables:
+      pr:
+        mip: Amon
+        era5_name: total_precipitation
+        era5_freq: monthly
```

Comment:
These two items are not needed for a successful run. However,  `era5_name` gives useful information. Could you please add a  note to the description section something like the variable pr is total precipitation in era5 archive. This will help users.

Reply:
Are you sure? I'd assume they are needed in 
ESMValTool/esmvaltool/diag_scripts/cmorizers/era5.py
e.g.:
`if info['diagnostic'] == "monthly":`
?
But I can test it.

Reply:
You are right, it works without